### ISSUE TITLE
F3 + ESC (Pause Game) crashes minecraft

### ISSUE THREAD

**Issue by dadoirie** [magicus/ModSettings#14]
the title says everything ...
here is the uploaded log
as stated on the log all the mods are disabled and only fabric api, mod menu and mod settings are loaded
[https://mclo.gs/rI0WGgk](https://mclo.gs/rI0WGgk)

**Comment by magicus** [comment]
```
	java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
	at net.fabricmc.fabric.impl.client.screen.ButtonList.translateIndex(ButtonList.java:130) ~[fabric-screen-api-v1-1.0.10 d882b91560-713a1bc331c59ce.jar:?]
	at net.fabricmc.fabric.impl.client.screen.ButtonList.get(ButtonList.java:48) ~[fabric-screen-api-v1-1.0.10 d882b91560-713a1bc331c59ce.jar:?]
	at net.fabricmc.fabric.impl.client.screen.ButtonList.get(ButtonList.java:33) ~[fabric-screen-api-v1-1.0.10 d882b91560-713a1bc331c59ce.jar:?]
	at se.icus.mag.modsettings.gui.MenuScreensChanger.injectModSettingsButton(MenuScreensChanger.java:62) ~[modsettings-1.1.0 1.18.1.jar:?]
	at se.icus.mag.modsettings.gui.MenuScreensChanger.postGameMenuScreenInit(MenuScreensChanger.java:27) ~[modsettings-1.1.0 1.18.1.jar:?]
```

**Comment by magicus** [comment]
I have failed to reproduce this.

I'm not sure how the game ever ended up in this state, but apparently there were no buttons in the in-game menu, which caused a crash when trying to get the first button. I have fixed so this situation will not result in a crash.

**Comment by magicus** [comment]
Ah,

**Comment by magicus** [comment]
Will be fixed in 2.0.0.
